cfr.json

```diff
@@ -445,6 +445,6 @@
         }
       ]
     },
-    "targeting": "source == 'startup' && !isMajorUpgrade && browserSettings.update.channel in ['release', 'beta'] && version|versionCompare('100.0.1') >= 0 && 'network.cookie.cookieBehavior'|preferenceValue == 4 && !'privacy.restrict3rdpartystorage.rollout.enabledByDefault'|preferenceIsUserSet && !'browser.privatebrowsing.autostart'|preferenceValue && !'browser.search.param.google_channel_us'|preferenceValue('')|regExpMatch('^[ntc]us5$') && !'browser.search.param.google_channel_row'|preferenceValue('')|regExpMatch('^[ntc]row5$') && [userId, 'tcp-spotlight-opt-in-rollout']|bucketSample(0, 2500, 10000)"
+    "targeting": "source == 'startup' && !isMajorUpgrade && 'app.shield.optoutstudies.enabled'|preferenceValue && browserSettings.update.channel in ['release', 'beta', 'aurora'] && version|versionCompare('100.0.1') >= 0 && 'network.cookie.cookieBehavior'|preferenceValue == 4 && !'privacy.restrict3rdpartystorage.rollout.enabledByDefault'|preferenceIsUserSet && !'browser.privatebrowsing.autostart'|preferenceValue && !'browser.search.param.google_channel_us'|preferenceValue('')|regExpMatch('^[ntc]us5$') && !'browser.search.param.google_channel_row'|preferenceValue('')|regExpMatch('^[ntc]row5$') && currentDate|date - profileAgeCreated >= 10800000"
```

Comment:
My understanding rollout don't show up in about:studies. Curious if  'app.shield.optoutstudies.enabled' check valid here as it's not a experiment but rollout.  Will browsing setting of ['do not recommend features/extension' pref ](https://searchfox.org/mozilla-central/rev/301e9545863b14ff85aa110fabd4274da4593f9d/browser/components/enterprisepolicies/Policies.jsm#722)makes more sense.